1-installation/index.md

```diff
@@ -35,82 +48,158 @@ Hint: don't worry if you make a typo when writing commands into the <span class=
+<div class="os-switch">
+<span class="osx">
+<code class="highlighter-rouge">cd</code> stands for "<strong>c</strong>hange <strong>d</strong>irectory" (directory is just another name for a folder). You can use <code class="highlighter-rouge">pwd</code> to check whether you really moved into the right place. The result you should now see is:
+
+<div class="highlighter-rouge"><pre class="highlight"><code>$ pwd
+/Users/codecat/Desktop
+</code></pre></div>
+
+As before, you can now also display a list of files and folders on your desktop with a familiar command:
+
+<div class="highlighter-rouge"><pre class="highlight"><code>$ ls
+</code></pre></div>
+
+</span>
+<span class="wins">
+<code class="highlighter-rouge">cd</code> with aditional arguments means "<strong>c</strong>hange <strong>d</strong>irectory" (remember before we just wanted to print the path of the current directory and we didn't pass any aditional arguments to the command). Now we moved to the directory  <code class="highlighter-rouge">Desktop</code> directory because we passed additional "Desktop" word to the command. The result you should now see is:
 
-	$ pwd
-	/Users/codecat/Desktop
+<div class="highlighter-rouge"><pre class="highlight"><code>C:\Users\codecat\Desktop
+</code></pre></div>
 
 As before, you can now also display a list of files and folders on your desktop with a familiar command:
 
-	$ ls
+<div class="highlighter-rouge"><pre class="highlight"><code>> dir
+</code></pre></div>
+</span>
+</div>
 
 Now, let's make a new folder on your Desktop, using only text commands. Type in the following command:
 
 	$ mkdir cat-folder
 
-`mkdir` is just a super short version of "**m**a**k**e **dir**ectory" and `cat-folder` is the name of the folder we want to create (you could choose any other name here). 
+`mkdir` is just a super short version of "**m**a**k**e **dir**ectory" and `cat-folder` is the name of the folder we want to create (you could choose any other name here).
 
 Now, check you Desktop and you'll see that a new folder with the name `cat-folder` just appeared! You can try and make another folder if you want.
 
 To clean up the newly created empty folder, you can use the command:
 
-	$ rm -r cat-folder
-
-`rm` stand for "**r**e**m**ove". Be careful when using this command, there's no undo, so make sure you're only deleting files and folder you don't need anymore.
+<div class="os-switch">
+<span class="osx">
+<div class="highlighter-rouge"><pre class="highlight"><code>$ rm -r cat-folder
+</code></pre></div>
+</span>
+<span class="wins">
+<div class="highlighter-rouge"><pre class="highlight"><code>> rmdir /s cat-folder
+</code></pre></div>
+</span>
+</div>
+<code class="highlighter-rouge"><span class="os-switch"><span class="osx">rm</span><span class="wins">rmdir</span></span></code> stands for "**r**e**m**ove". Be careful when using this command, there's no undo, so make sure you're only deleting files and folder you don't need anymore.
```

Comment:
Include the explanation in the os switcher (on Win it should be: rmdir stands for “remove directory")